src/components/Header/Header.js

```diff
@@ -1,13 +1,42 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import HeaderMenu from "./HeaderMenu";
+import HeaderIcon from "./HeaderIcon/HeaderIcon";
 import styled from "styled-components";
+import variables from "styles/variables";
```

Comment:
```suggestion
import styled from "styled-components";
import { Link } from "react-router-dom";
import HeaderMenu from "./HeaderMenu";
import HeaderIcon from "./HeaderIcon/HeaderIcon";
import variables from "styles/variables";
```

import 순서는 이렇게 하는게 좋겠습니다.
styled 도 라이브러리에서 가져온 것이기 때문에 css 라기 보다는 라이브러리에 가깝겠네요

Reply:
네 반영했습니다!